src/eclingo/control.py

```diff
@@ -16,12 +16,8 @@ def __init__(self, control, config=None):
         control.configuration.solve.project = "auto,3"
         control.configuration.solve.models = 0
         self.control = control
-        # else:
-        #     self.project    = None
-        #     self.max_models = 1
-        #     self.control = internal_states.InternalStateControl(['0', '--project'])
         if config is None:
-            config = AppConfig()
+            config = AppConfig(semantics="c19-1", use_reification=True)
```

Comment:
Eventually, the argument use_reification will not be needed. I'll clean all that up as a new issue in the future.